packages/edit-post/src/components/preferences-modal/index.js

```diff
@@ -95,6 +111,14 @@ export default function EditPostPreferencesModal() {
 								'Customize options related to the block editor interface and editing flow.'
 							) }
 						>
+							<EnableFeature
+								featureName="distractionFree"
```

Comment:
Should users who have 'reducedUI' enabled have 'distractionFree' enabled from the start? If so there should be a migration of the setting to the new name.

It can be added to this function - https://github.com/WordPress/gutenberg/blob/trunk/packages/preferences-persistence/src/migrations/preferences-package-data/index.js#L7

Something like:
```js
export default function convertPreferencesPackageData( data ) {
    data = convertReducedUIToDistractionFree( data );
	data = convertComplementaryAreas( data );
    return data;
}
```

And then `convertReducedUIToDistractionFree` would need to be implemented with unit tests. It shouldn't be too complicated. From my memory, `data` will be in this kind of structure:
```js
{
    'core/edit-post': {
        'reducedUI': true // or false
        // ... other preference values
    },
    // ... other editors
}
```

and would need to be converted to this:
```js
{
    'core/edit-post': {
        'distractionFree': true // or false
        // ... other preference values
    },
    // ... other editors
}
```

Reply:
We should look to remove `reducedUI` from the persisted data at some point later. No hurry to do that though.

Reply:
I'd migrate and avoid an orphaned option, reduced UI always felt half done.

Reply:
@mtias the orphaned option is a non issue. Like Dan mentioned it can simply be removed. Migrating is a very surprising UI change landing with no warning. Plus we cannot say that there is a  one to one correspondence between reduced UI and DFM and that 100% of users choosing one would choose the other. I think precisely because reduced UI felt half done it should be removed and  _not_ migrate users :)